pkg/controller/node/node_controller.go

```diff
@@ -65,11 +66,13 @@ const (
 	masterPoolName = "master"
 )
 
-var nodeUpdateBackoff = wait.Backoff{
-	Steps:    5,
-	Duration: 100 * time.Millisecond,
-	Jitter:   1.0,
-}
+var (
+	nodeUpdateBackoff = wait.Backoff{
```

Comment:
you have a public `NodeUpdateBackoff` in `pkg/constants` now.  Can you drop this one?

Reply:
and it looks like you do drop it in 652de252.  Maybe reshuffle your hunks between commits or squash down to one to avoid touching something in one commit and then removing it in the second?